pkg/chrootuser/user_linux.go

```diff
@@ -214,7 +218,7 @@ func lookupAdditionalGroupsForUIDInContainer(rootdir string, userid uint64) (gid
 	// Get the username associated with userid
 	username, _, err := lookupGroupForUIDInContainer(rootdir, userid)
 	if err != nil {
-		return nil, err
+		return nil, ErrNoSuchUser
```

Comment:
I don't like eating `err` here but I don't think we have an easy way around this without a big refactor of `lookupGroupForUIDInContainer`